### ISSUE TITLE
[Feature] Option to emit tabular output of tests

### ISSUE THREAD

**Issue by timheuer** [test-summary/action#2]
It is valuable to immediately see in the summary a list of the tests that passed/failed and individual timing for each

Inspiration 😛
![image](https://user-images.githubusercontent.com/4821/167472154-baa9b5c6-c853-4e72-a5f6-a9727f70f47c.png)

**Comment by ethomson** [comment]
Agreed. Right now you get the table with failures, but I would like to give you an option to show the full table. I was worried about the size (GitHub Actions test summaries have a length cap) but they increased that maximum so we should comfortably fit most test results (but [not all!](https://github.com/test-summary/action/issues/5)). 😁
